tests/ert_tests/status/test_evaluator_tracker.py

```diff
@@ -278,9 +313,12 @@ def test_tracking_progress(
     All events leading up to the final update event and end event, are
     discarded. The test asserts that the state of the world is correct only for
     the final update event."""
-    brm = BaseRunModel(None, None)
+    if issubclass(run_model, ERT3RunModel):
+        brm = run_model()
+    else:
+        brm = run_model(None, None)
```

Comment:
Because ERT3RunModel has different number of parameters than BaseRunModel..

Reply:
Yes, so if the parameter itself is ERT3RunModel() or BaseRunModel(None, None) depending on the test case, and the test would receive an instance of the run model, instead of creating the instance in the test itself? :slightly_smiling_face:

Reply:
So something like: 
```python
pytest.param(
            BaseRunModel(None, None),
            [
```
And then in the test itself just use `run_model` instead of `brm`
```suggestion
```